oci/main.py

```diff
@@ -43,15 +43,66 @@ def run():
              'under "ENGINE_VERSION"',
         default='master')
 
+    stage_parser = subparsers.add_parser(
+        'run',
+        help='run stage for a change')
+    stage_parser.set_defaults(command=run_stage)
+
+    stage_parser.add_argument(
+        '--stage',
+        '-s',
+        help='stage name')
+
+    stage_parser.add_argument(
+        'change',
+        help='Gerrit change number')
+
     args = parser.parse_args()
 
     logging.basicConfig(
         level=logging.DEBUG if args.debug else logging.WARNING,
         format="%(asctime)s %(levelname)-7s [%(name)s] %(message)s")
-
     args.command(args)
 
 
+def run_stage(args):
```

Comment:
For consistency with other code, lets call this run()

Reply:
Well the thing is the name of the main function is run().

https://github.com/nirs/oci/blob/23a56466ee97c0e2e6a1da4a5ab5f42695f76f04/oci/main.py#L11-L13

Reply:
So we can use run_stage_parser.

Reply:
Done.